### PR TITLE
Windows packaging job

### PR DESCRIPTION
**Pull request by esteve**

### CODE REVIEW

File: ros2_batch_job/packaging.py [hunk 1]
```diff
@@ -28,10 +29,12 @@ def main(sysargv=None):
 
 def build_and_package(args, job):
     # ignore ROS 1 bridge package for now
-    ros1_bridge_path = os.path.join(args.sourcespace, 'ros1_bridge')
+    ros1_bridge_path = os.path.join(args.sourcespace, 'ros2', 'ros1_bridge')
```
Comment (esteve) [inline ros2_batch_job/packaging.py]: I had to change this to make this work locally, but might break the buildfarm, I can change it.

Comment (dirk-thomas) [inline ros2_batch_job/packaging.py]: My custom repos file (https://gist.githubusercontent.com/dirk-thomas/441b614d57e2da2e0326/raw/4a62cebf7b45c20e5bfd713ee31e9b68c5ecfb78/gistfile1.txt) had the `ros1_bridge` in a different folder. But with ros2/examples#54 merged this should be the correct location.

+1

**Comment by tfoote** [comment]
+1

**Comment by wjwwood** [comment]
+1

**Comment by esteve** [comment]
I need to test a workaround for copying the OpenCV DLLs and I'll merge it afterwards.

**Comment by esteve** [comment]
Merging now. The buildfarm generated a zip file for Windows with the OpenCV DLLs:

http://ci.ros2.org/view/packaging/job/ros2_packaging_windows/4/

**Pull request merged**